source/extensions/tracers/datadog/dd.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+namespace datadog {
+namespace tracing {} // namespace tracing
+} // namespace datadog
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace Datadog {
+
+namespace dd = ::datadog::tracing;
```

Comment:
let's put this using declaration in the .cc files that  need it.

Reply:
I removed the alias. It's unnecessary.